nextage_tutorials/model/NextageKinect.xacro

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<robot name="NextageOpen" xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find nextage_description)/urdf/NextageOpen.urdf" />
+
+  <link name="odom" />
+  <joint name="odom_to_WAIST" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0.0 0.97" />
+    <parent link="odom" />
+    <child link="WAIST" />
+  </joint>
+
+  <!-- copied from   fetch_description/robots/fetch.urdf -->
+
+  <link name="head_camera_rgb_frame" />
+  <joint name="head_camera_rgb_joint" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0.02 0" />
+    <parent link="HEAD_JOINT1_Link" />
+    <child link="head_camera_rgb_frame" />
+  </joint>
+  <link name="head_camera_rgb_optical_frame" />
+  <joint name="head_camera_rgb_optical_joint" type="fixed">
+    <origin rpy="-1.57079632679 0 -1.57079632679" xyz="0 0 0" />
+    <parent link="head_camera_rgb_frame" />
+    <child link="head_camera_rgb_optical_frame" />
+  </joint>
+  <link name="head_camera_depth_frame" />
+  <joint name="head_camera_depth_joint" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0.045 0" />
+    <parent link="HEAD_JOINT1_Link" />
+    <child link="head_camera_depth_frame" />
+  </joint>
+  <link name="head_camera_depth_optical_frame" />
+  <joint name="head_camera_depth_optical_joint" type="fixed">
+    <origin rpy="-1.57079632679 0 -1.57079632679" xyz="0 0 0" />
+    <parent link="head_camera_depth_frame" />
+    <child link="head_camera_depth_optical_frame" />
+  </joint>
+
+  <!-- copied from fetch_gazebo/robots/fetch.gazebo.xacro -->
+
+  <!-- Primesense Carmine 1.09 -->
+  <gazebo reference="head_camera_rgb_frame">
+    <sensor type="depth" name="camera">
+      <always_on>true</always_on>
+      <update_rate>15.0</update_rate>
+      <camera>
+        <horizontal_fov>1.047197</horizontal_fov>
+        <image>
+          <!-- openni_kinect plugin works only with BGR8 -->
+          <format>B8G8R8</format>
+          <width>640</width>
+          <height>480</height>
+        </image>
+        <clip>
+          <near>0.05</near>
+          <far>50</far>
+        </clip>
+      </camera>
+      <plugin name="head_camera_controller" filename="libgazebo_ros_openni_kinect.so">
+        <baseline>0.1</baseline>
+        <alwaysOn>true</alwaysOn>
+        <updateRate>15.0</updateRate>
+        <cameraName>head_camera</cameraName>
+        <imageTopicName>/head_camera/rgb/image_raw</imageTopicName>
+        <cameraInfoTopicName>/head_camera/rgb/camera_info</cameraInfoTopicName>
+        <depthImageTopicName>/head_camera/depth_registered/image_raw</depthImageTopicName>
```

Comment:
As [nextage_gazebo uses turtlebot_description](https://github.com/tork-a/rtmros_nextage/blob/0.8.5/nextage_calibration/models/sensor_kinect.xacro#L3) and [turtlebot_description outputs to depth/image_raw](https://github.com/turtlebot/turtlebot/blob/feb176b10f13bcdc517e7f305c39327dac6388f0/turtlebot_description/urdf/turtlebot_gazebo.urdf.xacro#L26), I misunderstood that this topic is unregistered and [registered again](https://github.com/ros-drivers/freenect_stack/blob/0.4.3/freenect_launch/launch/freenect.launch#L109).
I think perhaps this is wrong and `depth_registered/points` lost color.
Also, I'm suspicious about TF relation declared [here](https://github.com/ros-drivers/freenect_stack/blob/0.4.3/freenect_launch/launch/freenect.launch#L131).

Reply:
could you creaet PR for your proposal?